.pylintrc

```diff
@@ -0,0 +1,588 @@
+[MASTER]
```

Comment:
Is this just straight generated, or did you make any changes?

Reply:
Almost straight generated default.
I increased the max line length from 100 to 125.

Reply:
OK. How compatible is the rc file between version? Like if we start using a new pylint do we need to regenerate?

Reply:
I'm not sure. But you raise a good point. We've seen pylint failures because of changes in versions in the past.
I'll make a change to pin to a specific pylint version. Maybe yamllint as well.

Reply:
@jayfurmanek 
I pinned all of the test package versions.

Reply:
Thanks. We'll likely want to revisit the rc when we update the pin. We'll see how it goes.

Reply:
We may have to regenerate between versions, but that should be a conscious decision now.